### PR TITLE
Fix intent resolving behavior for Android 11

### PR DESCRIPTION
**Pull request by warwicklol14**
Fixes : #3067 

Signed-off-by: Varun Paul kjvpaul@tutanota.com

### Pull Request Checklist

- [X] Changes has been tested on an Android device or Android emulator with API 21
- [ ] UI change has been tested on both light and dark themes
- [X] Pull request is based on the develop branch
- [X] Pull request updates [CHANGES.md](https://github.com/vector-im/element-android/blob/develop/CHANGES.md)
- [ ] Pull request includes screenshots or videos if containing UI changes
- [X] Pull request includes a [sign off](https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.md#sign-off)

### CODE REVIEW

File: vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailFragment.kt [hunk 1]
```diff
@@ -595,9 +596,9 @@ class RoomDetailFragment @Inject constructor(
                 addFlags(Intent.FLAG_GRANT_READ_URI_PERMISSION or Intent.FLAG_ACTIVITY_NEW_TASK)
             }
 
-            if (intent.resolveActivity(requireActivity().packageManager) != null) {
```
Comment (bmarty) [inline vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailFragment.kt:598]: What about the 2 other usages of `resolveActivity` ?

**Review by bmarty — COMMENTED** [review]
Thanks for the PR, look good to me. One question though.

File: vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailFragment.kt (hunk 1 above)
Comment (warwicklol14) [reply vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailFragment.kt:598]: I did not change those as I don't have any idea how they're being used. I see that one usage of it for VectorWebViewClient and the other usage is for crypto. I honestly have no idea about crypto. I've been trying to learn the mvrx and epoxy library more so I can understand the overall code of this app better. If you could let me know how to test out the other usages of `resolveActivity`, I would love to help out by digging into it more.

Comment (bmarty) [reply vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailFragment.kt:598]: Applying the same fix at the 2 other places should be fine. Can you do that please?

**Pull request closed without merging**